lib/librato-services/helpers/snapshot_helpers.rb

```diff
@@ -12,7 +12,10 @@ def self.sample_snapshot_payload
             :snapshot => {
               :entity_name => "App API Requests",
               :entity_url => "https://metrics.librato.com/instruments/1234?duration=3600",
-              :image_url => "http://snapshots.librato.com/instruments/12345abcd.png"
+              :image_url => "http://snapshots.librato.com/instruments/12345abcd.png",
+              :user_email => "portal-dev@librato.com",
```

Comment:
Mike it hasn't been mentioned. Working on a spec here: https://docs.google.com/a/librato.com/document/d/1VzeLPthNpL4rGaW19bjvKmBRC0MlUZJGIqxncfdRxmI/edit?usp=sharing